### PR TITLE
Home/Rules/Partial for NIPD 2019

### PR DESCRIPTION
**Pull request by tabascq**
Content already reviewed by Michael; you can certainly look if you want but it isn't strictly necessary to do so.

### CODE REVIEW

File: ServerCore/Pages/Resources/nipd2019/FAQPartial.cshtml [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+
+    <h3>
+        I'm not that good at solving puzzles and some of these puzzles look hard. Can I still participate?
+    </h3>
+    <p>
+        Yes. As in any event like this, some teams and some players will be very dedicated. But the event is designed so that less dedicated teams will also have fun. It's also OK to have teams that consist of some hard core participants and some casual participants. There will be a variety of puzzles so there's a good chance there will be puzzles that you enjoy.
+    </p>
+
+    <h3>
+        I have special needs and/or require the use of assistive technology. Can I still participate?
```
[the remark this answers is not in the document]
Comment (tabascq) [reply ServerCore/Pages/Resources/nipd2019/FAQPartial.cshtml]: I pulled it.

File: ServerCore/Pages/Resources/nipd2019/HomePartial.cshtml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+            </p>
+
+            <p>
+                The event starts at <b>11AM in the Building 99 lobby</b>. Answer submission will close at <b>5PM</b>. We will have answers posted in the Building 99 lobby immediately afterwards.
+            </p>
+        </div>
+
+        <div class="pd2019-container-card">
+            <h3>New to puzzles?</h3>
+            <p>You might have seen a jigsaw puzzle, a crossword puzzle, or a Sudoku at some point in your life. But wait, there's more! Puzzleday puzzles <span style="text-decoration: line-through">sometimes</span> always dial up the creativity, so you might see some brain-twisty, mind-bendy puzzles... Check out <a asp-page="/EventSpecific/Samples">our sample puzzles</a>!</p>
```
[the remark this answers is not in the document]
Comment (tabascq) [reply ServerCore/Pages/Resources/nipd2019/HomePartial.cshtml]: That link is OK, no subdirectories were used. Tools, Encodings, etc. work the same way.

**Comment by tabascq** [comment]
Updated, thanks for the catches!

File: ServerCore/Pages/Resources/nipd2019/RulesPartial.cshtml [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+        <li>There are ONLY two ways that you might be instructed to travel.</li>
+        <ul>
+            <li>The puzzle has explicit instructions to travel. For example, your team may be scheduled for a live event at a specified location.</li>
+            <li>You submit an answer and the response from the website gives you explicit instructions to travel. If you have an answer that looks like a travel instruction, enter it into the submission box in full and only travel if the response says so.</li>
+        </ul>
+        <li>
+            You must not enter any person's private office during the course of the event unless it is the office of a member of your team.
+        </li>
+        <li>
+            If any travel is required, it will take place East of SR-520, South of NE 40th Street, and North of Bel-Red Road. Do not enter the following buildings unless your personal office is in them: 26, 30, 34, 50, and 87. Do not enter any construction areas.
```
[the remark this answers is not in the document]
Comment (tabascq) [reply ServerCore/Pages/Resources/nipd2019/RulesPartial.cshtml]: Are we sending people to your puzzle's location?

File: ServerCore/Pages/Resources/nipd2019/RulesPartial.cshtml [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
```
Comment (morganbr) [inline ServerCore/Pages/Resources/nipd2019/RulesPartial.cshtml]: Do we have an up to date list of buildings that includes West campus?

File: ServerCore/Pages/Resources/nipd2019/FAQPartial.cshtml [hunk 5]
```diff
@@ -0,0 +1,155 @@
...
+    </h3>
+    <p>
+        If you don't understand how to solve a puzzle, contact puzzle control. We may give you a hint or clarify the instructions. Aside from that, there are enough puzzles that you probably won't be stuck on all of them.
+    </p>
+
+    <h3>
+        When do we get hints?
+    </h3>
+    <p>
+        Puzzle control will provide hints during the event. @*in two ways. First, we will distribute sets of hints to all teams at the same time by email at least once during the event. Second, we will sometimes offer hints to teams when they send email to puzzle control. The decision of what hints to provide is completely at the discretion of puzzle control. For example, towards the end of the day we will be liberal with providing hints to teams that clearly have no chance of winning so that they can continue to have fun. We are more likely to give a strong hint to a puzzle that no team has solved than one that's been solved by just one team. We would also be more likely to give a strong hint to a puzzle that has already been solved by all the top teams. So don't get discouraged when we give hints to puzzles you've already solved.*@
```
Comment (morganbr) [inline ServerCore/Pages/Resources/nipd2019/FAQPartial.cshtml]: Please remove commentted text

File: ServerCore/Pages/Resources/nipd2019/FAQPartial.cshtml [hunk 6]
```diff
@@ -0,0 +1,155 @@
...
+    </h3>
+    <p>
+        Not having fun. If you're not having fun working on a particular puzzle, switch to another one. Don't worry, you're unlikely to run out of puzzles.
+    </p>
+
+    <h3>
+        What if we're really stuck?
+    </h3>
+    <p>
+        If you don't understand how to solve a puzzle, contact puzzle control. We may give you a hint or clarify the instructions. Aside from that, there are enough puzzles that you probably won't be stuck on all of them.
```
Comment (morganbr) [inline ServerCore/Pages/Resources/nipd2019/FAQPartial.cshtml]: Should this say to use the links on the submission page to contact puzzle control?

File: ServerCore/Pages/Resources/nipd2019/FAQPartial.cshtml [hunk 7]
```diff
@@ -0,0 +1,155 @@
...
+    </h3>
+    <p>
+        No. 😇
+    </p>
+
+    <h3>
+        Do I have to register to participate?
+    </h3>
+    <p>
+        Yes. If you miss the registration deadline, send mail to puzzle control at <a href="mailto:puzzhunt@microsoft.com">puzzhunt</a>.
```
Comment (morganbr) [inline ServerCore/Pages/Resources/nipd2019/FAQPartial.cshtml]: What would we do if somebody emailed us?

File: ServerCore/Pages/Resources/nipd2019/FAQPartial.cshtml [hunk 8]
```diff
@@ -0,0 +1,155 @@
...
+    </h3>
+    <p>
+        Yes. As in any event like this, some teams and some players will be very dedicated. But the event is designed so that less dedicated teams will also have fun. It's also OK to have teams that consist of some hard core participants and some casual participants. There will be a variety of puzzles so there's a good chance there will be puzzles that you enjoy.
+    </p>
+
+    <h3>
+        How do I contact puzzle control?
+    </h3>
+    <p>
+        Send email to <a href="mailto:puzzhunt@microsoft.com">puzzhunt</a>. We will also provide you with a telephone number to contact us at during the event.
```
Comment (morganbr) [inline ServerCore/Pages/Resources/nipd2019/FAQPartial.cshtml]: Will we actually provide a phone number?

File: ServerCore/Pages/Resources/nipd2019/RulesPartial.cshtml (hunk 4 above)
Comment (tabascq) [reply ServerCore/Pages/Resources/nipd2019/RulesPartial.cshtml]: Nope, not going to bother since this statement is technically true as we know it.

**Pull request merged**